modules/stanford_profile_styles/lib/scss/component/node/stanford_page.layout.full-width.scss

```diff
@@ -24,6 +24,18 @@ body.front {
       max-width: calc(100vw - 200px);
     }
   }
-
 }
 
+.main-region {
+  &.flex-10-of-12 {
```

Comment:
This defeats the purpose of 10 of 12 columns. The better solution is to update the full width layout itself. We might not be able to achieve the same solution you have here, but changing it to a "centered" container with default width would match other pages.

See what you can do on `/admin/structure/layouts` page.

Reply:
Ok. Thank you for the tip. I have it working and undid the change above too.  I'll update the Prs.